.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly consider adding a descriptive Pr title and remember Pull Request should have a _**descriptive title**_ telling you what changes are inside and this too will make your pr descriptive. The current title  `Adding .gitignore, linters, README, and HTML/CSS files` Should change to something that easily speaks of the project. e.g **_Hello Microverse Project_**. 

- [ ]  kindly consider adding a Pr description and remember Pull Request should have a short summary describing introduced changes in more detail this will make it more professional and descriptive. You could use bullets if you please.

Reply:
### **Hello Microverse Project.**

In this Hello World project, I made the following changes:

- Added a .gitignore file to exclude node_modules.
- Set up a linter for HTML and CSS and added a linters file.
- Added a descriptive README file based on the provided template.
- Added an HTML file with a header that says "Hello Microverse!" using h1 tag.
- Added a CSS file with a class to style the header in my favorite color.
- Added a test.md file to .gitignore tp ensure it is ignored by git.